python/deepsulci/sulci_labeling/capsul/labeling.py

```diff
@@ -139,6 +152,13 @@ def _run_process(self):
 
         graph['label_property'] = 'label'
         # save graph
-        aims.write(graph, self.labeled_graph,
+        aims.write(graph, self.labelled_graph,
                    options={"save_only_modified": False})
+        # since some cuts have been made, we need to update the voronoi and attributes that depend on it.
+        if summary['cuts'] != 0:
+            soma.subprocess.call(['AimsFoldsGraphThickness.py', '-i', self.labelled_graph,
```

Comment:
I use soma.subprocess to call directly the command, I thought it was the simpler way but I don't known if it is way to do it in capsul process.

Reply:
It's OK here. No need to start another process, anyway the parameters are different (used on the labelled graph, not saving the maps...)